src/components/application_manager/src/app_service_manager.cc

```diff
@@ -80,6 +80,16 @@ smart_objects::SmartObject AppServiceManager::PublishAppService(
     return smart_objects::SmartObject();
   }
 
+  if (manifest.keyExists(strings::service_name)) {
+    existing_service =
```

Comment:
@jacobkeeler cant see if this function is used in further code.
 I would prefer removing this temporary, probably NULL pointer variable